packages/webdriver/src/request.ts

```diff
@@ -98,36 +119,43 @@ export default class WebDriverRequest extends EventEmitter {
         /**
          * if the environment variable "STRICT_SSL" is defined as "false", it doesn't require SSL certificates to be valid.
          */
-        requestOptions.https.rejectUnauthorized = !(
+        requestOptions.https!.rejectUnauthorized = !(
```

Comment:
worried about using `non-null` assertions, not just here but across the PR

Reply:
this is hardcoded in line 73

Reply:
will it work if you change type in line 73 to `got.Options & { https: HTTPSOptions }` ?

Reply:
same for headers if it works for `https`